salt/migrations/2-3/etcd/pre-reboot.sls

```diff
@@ -0,0 +1,24 @@
+{% set roles = salt['grains.get']('roles', []) %}
+{% set has_etcd_role = ("etcd" in roles) %}
+
+{% if not has_etcd_role %}
+  # make sure there is nothing left in /var/lib/etcd
+
+cleanup-old-etcd-stuff:
+  cmd.run:
+    - name: rm -rf /var/lib/etcd/*
```

Comment:
Wouldn't it be more *"salty"* to use a `salt.states.file.absent`?

Reply:
As far as I know there's no way to replicate this exact command, `file.absent` will do a `rm -rf /var/lib/etcd/`, removing the `etcd` folder too.

Reply:
:+1: